src/store/actions/api.actions.js

```diff
@@ -246,40 +488,39 @@ export const saveEndpoint = (pathname, api) => (dispatch) => {
     // substitude updated list of plugins
     const preparedApi = R.set(R.lensPath(['plugins']), preparedPlugins, api);
 
-    return client.post('apis', preparedApi)
-    .then((response) => {
-        // dispatch(saveEndpointSuccess(JSON.parse(response.config.data)));
-        dispatch(saveEndpointSuccess());
-        dispatch(openResponseModal({
-            status: response.status,
-            message: 'Successfuly saved',
-            statusText: response.statusText,
-            redirectOnClose: () => (history.push('/')),
-        }));
-    })
-    .catch((error) => {
-        if (error.response) {
+    return client.put(`apis${pathname}`, preparedApi)
```

Comment:
async/await here too

Reply:
Yeap, @rafaeljesus , this is done is separate **PR**[https://github.com/hellofresh/janus-dashboard/pull/39](https://github.com/hellofresh/janus-dashboard/pull/39)